moped-editor/src/views/projects/projectView/ProjectNameForm.js

```diff
@@ -136,18 +137,18 @@ const ProjectNameForm = ({
             container
             direction="row"
             alignItems="center"
+            alignContent="center"
             xs={12}
             sm="auto"
             sx={(theme) => ({
               minWidth: theme.spacing(12),
             })}
           >
-            <IconButton type="submit" disabled={!isDirty || loading}>
-              <Icon>check</Icon>
-            </IconButton>
-            <IconButton onClick={handleCancelClick} disabled={loading}>
-              <Icon>close</Icon>
-            </IconButton>
+            <ProjectSummaryIconButtons
+              handleClose={handleCancelClick}
+              disabledCondition={!isDirty}
```

Comment:
This is just a uninformed question on my part -- what does `isDirty` signify? Is that a form that has an edit but is unsaved?